tests/components/august/mocks.py

```diff
@@ -35,14 +54,49 @@ def action(self):
         return self._action
 
 
+class MockAugustLock(AugustLock):
```

Comment:
`AugustLock` is the entity class. Why inherit from that here?

Reply:
I cleaned this up as much as I could without changing other mocks.  There is a lot more to avoid mocking out parts of the component.  I'll do additional cleanup once I switch it over to DataUpdateCoordinator